src/ServiceProvider.php

```diff
@@ -59,7 +59,11 @@ public function register(): void
     public function boot(): void
     {
         $this->publishConfig();
-
+        
+        if ($this->isAgentDisabled()) {
+            return;
+        }
+        
```

Comment:
Some spaces need to be removed still.

```suggestion

        if ($this->isAgentDisabled()) {
            return;
        }

```